### PR TITLE
Adds cross-section generation lines to full-core serpent script

### LINKED ISSUE #17: Generate cross-sections for the full core model
This issue can be closed when the full core model is capable of generating the cross-sections to use in Moltres.

### PR DESCRIPTION
**Pull request by robfairh**
Fixes #17.

This PR modifies the following files:
* fullcore: adds cross-sections generation lines
* auxiliary.py: adds a function that plots standard-column model flux

### CODE REVIEW

File: auxiliary.py [hunk 1]
```diff
@@ -181,6 +153,46 @@ def plot_axial(data, vb, vc, vt):
...
+    plt.figure()
+    plt.step(z, val[0], where='post', label='thermal')
+    plt.step(z, val[1], where='post', label='fast')
+    plt.xlabel('z [cm]')
+    plt.ylabel(r'$\phi$')
+    plt.legend(loc="upper right")
+    plt.savefig(name, dpi=300, bbox_inches="tight")
+
+
+def plots_standardcolum():
```
Comment (smpark7) [inline auxiliary.py]: Is this a typo? Do you mean plots_standardcolumn()?

File: auxiliary.py [hunk 2]
```diff
@@ -181,6 +153,46 @@ def plot_axial(data, vb, vc, vt):
...
+    '''
```
Comment (smpark7) [inline auxiliary.py]: Use double quotes " here

**Review by smpark7 — COMMENTED** [review]
Looks mostly good. I'll merge after you settle the minor comments

File: auxiliary.py (hunk 1 above)
Comment (robfairh) [reply auxiliary.py]: you are certainly right

File: auxiliary.py (hunk 2 above)
Comment (robfairh) [reply auxiliary.py]: you are right once more

**Review by smpark7 — APPROVED** [review]
Looks good!

**Pull request merged**